source/WebApp/package-lock.json

```diff
@@ -9778,7 +9778,7 @@
       }
     },
     "regex-combiner": {
-      "version": "github:am11/regex-combiner#9a45822e2a1beaf95cb5881e69f0c6660a690ab7",
+      "version": "github:am11/regex-combiner#1fd650aa8dcec8d71e0101558bc181eed8b9ed08",

```

Comment:
I think it would be good to fix the dependency commit hash in `package.json` as well.
If you need to update your branch you will need to do a PR either way, and I think it's good to have `package.json` as the source of truth on dependency versions.